shared/variables/global.js

```diff
@@ -217,6 +219,10 @@ export const IOS_MI_PACKAGE_VERSION = '4.0.0-rc.1';
 export const IOS_PACKAGE = 'NAME: AgoraRtcEngine_iOS';
 export const IOS_VOICE_PACKAGE = 'AgoraRtcEngine_iOS_Voice_Beta';
 
+export const NATIVE = 'Native';
+export const RTC_NATIVE_SDK = `RTC ${NATIVE} SDK`;
+export const RTC_WEB_SDK = `RTC ${WEB} SDK`;
+
```

Comment:
We don't operate with the name RTC SDK though, do we? It's Voice SDK or Video SDK. Also, you probably need to keep the web version for the web page, and the native versions (specifying the platform) for each of the other pages using conditional text.

Reply:
Video SDK or Voice SDK, certainly not native. Please update any references to this terminology. 

If you must, Video SDK for <Android|iOS|etc.> However, as you are already in a doc that selects by platform, this does not really serve any purpose.